### PR TITLE
test: add unit tests for time-metrics util class.

### PR DESCRIPTION
**Pull request by ganesh-nithin**
# Why this change is required

This PR aims to add unit tests for TimeMetrics class.

# Describe your changes

- added tests for TimeMetrics class.
- fixed approval time calculation.

current - while calculating approval time we are checking at least one required reviewer approved the PR.
implemented - all the required reviewers should approve the PR to consider the PR as approved.

# How this change has been tested

Tested using jest test runner.

# Reference

image

# Checklist

- [x] I have performed a self review of my code and intend to submit as such
- [ ] I have added necessary explanations and inline comments for review
- [ ] I have considered updating necessary README or other documentations
- [x] I have added/modified necessary automated tests
- [ ] This includes a breaking changes and needs to be listed in release notes

### CODE REVIEW

File: server/src/use-cases/version-control/azure-devops/code-review/time-metrics/time-metrics.unit.test.js
Comment (ganesh-nithin) [inline server/src/use-cases/version-control/azure-devops/code-review/time-metrics/time-metrics.unit.test.js:1]: @sathish-thangavel , can these mock data stay in the specs, the data is less.

Comment (sathish-thangavel) [reply server/src/use-cases/version-control/azure-devops/code-review/time-metrics/time-metrics.unit.test.js:1]: Sure , works

File: server/src/use-cases/version-control/azure-devops/code-review/time-metrics/time-metrics.unit.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    const creationDate = new Date('2022-01-01T00:00:00Z');
+
+    const reviewers = {
+      1: { author: 'author1', isRequired: true, vote: VOTE.APPROVED },
+      2: { author: 'author2', isRequired: true, vote: VOTE.APPROVED },
+    };
+
+    const votesHistoryTimeline = [
+      { id: 1, author: 'author1', timeOfVote: new Date('2022-01-01T01:00:00Z'), vote: VOTE.APPROVED },
+      { id: 2, author: 'author2', timeOfVote: new Date('2022-01-01T02:00:00Z'), vote: VOTE.APPROVED },
```
Comment (sathish-thangavel) [inline server/src/use-cases/version-control/azure-devops/code-review/time-metrics/time-metrics.unit.test.js:81]: Can we add more test case covering combination of different votes ?

**Review by sathish-thangavel — CHANGES_REQUESTED** [review]

**Review by sathish-thangavel — APPROVED** [review]
LGTM!

**Pull request merged**